updates/ryan_2.sql

```diff
@@ -0,0 +1,33 @@
+
+CREATE TABLE IF NOT EXISTS `neg`.`selektion_angabe` (
+  `ID` INT NOT NULL AUTO_INCREMENT,
+  `Bezeichnung` VARCHAR(50) CHARACTER SET 'utf8mb4' COLLATE 'utf8mb4_unicode_ci' NOT NULL,
+  PRIMARY KEY (`ID`),
+  UNIQUE `unique_bezeichnung_selektion_angabe` (`Bezeichnung`)
+  )
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8mb4
+COLLATE = utf8mb4_unicode_ci;	
+
+/*Insert a default value*/
+INSERT INTO `neg`.`selektion_angabe` (ID,Bezeichnung) VALUES (-1,'-');
+INSERT INTO `neg`.`selektion_angabe` (ID,Bezeichnung) VALUES (1,'?');
+
+CREATE TABLE einzelbeleg_hatangabe (
+    ID int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
```

Comment:
This is basically correct, but please use the same way to describe the PRIMARY KEY as in selektion_angabe for consistency reasons.